src/Mvc/Mvc.Abstractions/src/Microsoft.AspNetCore.Mvc.Abstractions.WarningSuppressions.xml

```diff
@@ -6,7 +6,12 @@
       <argument>IL2026</argument>
       <property name="Scope">member</property>
       <property name="Target">M:Microsoft.AspNetCore.Mvc.ModelBinding.ModelMetadata.#ctor(Microsoft.AspNetCore.Mvc.ModelBinding.Metadata.ModelMetadataIdentity)</property>
-      <property name="Justification">This warning is left in the product so developers get an ILLink warning when trimming an app when Microsoft.AspNetCore.Mvc.ApiExplorer.IsEnhancedModelMetadataSupported=true.</property>
+    </attribute>
+    <attribute fullname="System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute">
+      <argument>ILLink</argument>
+      <argument>IL2026</argument>
+      <property name="Scope">member</property>
+      <property name="Target">M:Microsoft.AspNetCore.Mvc.ModelBinding.ModelMetadata.get_IsComplexType</property>
```

Comment:
```csharp
public bool IsComplexType => !IsConvertibleType && !IsParseableType;
```

Looks like IsComplexType will also need to be annotated with `RequiresUnreferencedCode` because it depends on `IsConvertibleType` and `IsParseableType` which both require unreferenced code.